packages/flutter_tools/test/src/base/file_system_test.dart

```diff
@@ -6,8 +6,29 @@ import 'package:file/memory.dart';
 import 'package:flutter_tools/src/base/file_system.dart';
 import 'package:test/test.dart';
 
+import '../common.dart';
+import '../context.dart';
+
 void main() {
-  group('file_system', () {
+  group('ensureDirectoryExists', () {
+    MemoryFileSystem fs;
+
+    setUp(() {
+      fs = new MemoryFileSystem();
+    });
+
+    testUsingContext('recursively creates a directory if it does not exist', () async {
```

Comment:
Due to line lenths? This is consistent with how we call `group()`, `test()`, and `testUsingContext()` elsewhere.